modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -236,5 +252,15 @@ function isFeatureIdDefined(value) {
   return value !== undefined && value !== null && value !== '';
 }
 
+function transformTileCoordsToWGS84(info, object, tile) {
+  return {
```

Comment:
This is potentially a very expensive operation, especially if you perform it on hover. Consider offering a callback that users can use to transform the coordinates only if they need it, similar to how query viewport features is implemented.

Reply:
Alternatively, you can delay the transformation to when `coordinates` is accessed:

```js
const feature = {properties, geometry: {type}};
Object.defineProperty(feature.geometry, 'coordinates', {
  get: () => {
    return transform(coordinates, tile)
  }
});
```